test/test_gem2rpm.rb

```diff
@@ -20,6 +20,26 @@ def setup
     end
   end
 
+  def test_show_message_for_message
+    expected = <<-'END'
+foo
+
+    END
+
+    Gem2Rpm.show_message("foo", nil, @out)
+    assert_equal expected, @out.string
+  end
+
+  def test_show_message_for_obj
+    obj = {
+      :a => "x",
+      :b => "y",
+    }
+    Gem2Rpm.show_message("foo", obj, @out)
+    refute_match(/\Afoo.*:a.*:b.*\Z/, @out.string)
```

Comment:
This would be more consistent with the rest of the test suite if you omitted the brackets.